docs/kr.tree

```diff
@@ -310,7 +310,8 @@
 	<toc-element toc-title="API reference">
 		<toc-element toc-title="Standard library (stdlib)" href="https://kotlinlang.org/api/latest/jvm/stdlib/"/>
 		<toc-element toc-title="Test library (kotlin.test)" href="https://kotlinlang.org/api/latest/kotlin.test/"/>
-		<toc-element toc-title="Coroutines (kotlinx.coroutines)" href="https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/"/>
+		<toc-element toc-title="Kotlin Gradle Plugins (kotlin-gradle-plugin)" href="https://kotlinlang.org/api/kotlin-gradle-plugin/"/>
```

Comment:
I have a question: 
From my perspective, it doesn't seem clear that there are actually two APIs: 
* Kotlin Gradle plugin
* Compose compiler Gradle plugin

What do you think if we use both of them in a tree?
Like:

```xml
<toc-element toc-title="Kotlin Gradle plugin" href="https://kotlinlang.org/api/kotlin-gradle-plugin/kotlin-gradle-plugin-api/"/>
<toc-element toc-title="Compose compiler Gradle plugin" href="https://kotlinlang.org/api/kotlin-gradle-plugin/compose-compiler-gradle-plugin/"/>

```

Reply:
And about positioning: I'd like to put it as the last items in a list.

Reply:
I don't know, we most probably add in the future reference for other Gradle plugins for our compiler plugins  (e.g. `allopen`). So I think I prefer current approach.

Reply:
Ok, let's try your approach and revisit this question later.
@zoobestik can you put the item lower and rename it to "plugins" not "Plugins"

Reply:
🆙